buildSrc/src/main/kotlin/kotlin-conventions.gradle.kts

```
@@ -27,6 +28,12 @@ tasks.withType<Test>().configureEach {
    }
 }
 
+tasks.withType<KotlinJvmTest>().configureEach {
+   outputs.cacheIf{
+      true
+   }
```

I recommend putting this on a single line. For example, see [Using the runtime API](https://docs.gradle.org/current/userguide/build_cache.html#using_the_runtime_api) of the Build Cache documentation.

```suggestion
   outputs.cacheIf { true }
```